src/test/test_api_v2.py

```diff
@@ -0,0 +1,175 @@
+"""
+Integration tests on API v2 methods.
+
+Version 2 adds Jinja2 templating support on stored queries.
+"""
+import unittest
+import requests
+import json
+import os
+
+from src.test.utils import save_test_docs
+
+# Use the mock auth tokens
+_ADMIN_TOKEN = 'admin_token'
+_INVALID_TOKEN = 'invalid_token'
+
+# Use the docker-compose url of the running flask server
+_URL = os.environ.get('TEST_URL', 'http://web:5000')
+_VERSION = 'v2'
+_API_URL = '/'.join([_URL, 'api', _VERSION])
+
+_HEADERS_ADMIN = {'Authorization': 'Bearer ' + _ADMIN_TOKEN, 'Content-Type': 'application/json'}
+
+
+class TestApi(unittest.TestCase):
```

Comment:
Out of an abundance of caution, it probably would not be a bad idea to retest the v1 endpoints that are just pulled though into the v2 API just to make sure that the routing does not get flubbed (this is the downside to the explicit format I pushed for). Keeping the tests dry might require a second test module that could be called by both the v1 and v2 tests (which would stay lean).

Reply:
It's true that it wouldn't be hard to leave out an endpoint or mistype the name. I'll see if i can rearrange things to reuse v1 tests, or making it backwards compatible, or something...